code/modules/mob/living/carbon/shock.dm

```diff
@@ -8,8 +8,7 @@
 	1	* src.getToxLoss() + 		\
 	1	* src.getFireLoss() + 		\
 	1	* src.getBruteLoss() + 		\
-	1	* src.getCloneLoss() + 		\
-	1	* src.halloss
+	1	* src.getCloneLoss()
```

Comment:
It's so you've got a bunch of variables in one place, that you can adjust at will for modifying the pain values of each type of damage.